README.md

```diff
@@ -1,26 +1,22 @@
-#DynamicCache
-
-A middleware layer for [ServiceWorkerWare](https://github.com/arcturus/serviceworkerware/) that will allow you to modify the content of a cache.
+# RawCache
+A middleware layer for [ServiceWorkerWare](https://github.com/gaia-components/serviceworkerware/) that will allow you to modify the content of a cache.
```

Comment:
I would like to add at the end:

... content of a cache via HTTP verbs.

wdyt?